Tests/FDBTests/TupleTests.swift

```diff
@@ -129,16 +129,26 @@ class TupleTests: XCTestCase {
         #endif
         let etalonTuple = FDB.Tuple(input)
         let packed = etalonTuple.pack()
-        let repacked = FDB.Tuple(from: packed).pack()
+        let repacked = try FDB.Tuple(from: packed).pack()
         XCTAssertEqual(packed, repacked)
     }
 
     // Fixes https://github.com/kirilltitov/FDBSwift/issues/10
-    func testNullEscapes() {
+    func testNullEscapes() throws {
         let packed = Bytes([0, 0, 0,]).pack()
-        let repacked = FDB.Tuple(from: packed).pack()
+        let repacked = try FDB.Tuple(from: packed).pack()
         XCTAssertEqual(packed, repacked)
     }
+    
+    func testUnpackSanity() {
+        for _ in 1...10000 {
+            do {
+                let _ = try FDB.Tuple(
+                    from: (1...UInt8.random(in: 10..<UInt8.max)).map { _ in UInt8.random(in: 0..<UInt8.max) }
```

Comment:
This might be unstable as I haven't yet implemented some codes. Need to watch this place.